.gitignore

```
@@ -3,15 +3,24 @@
 # User-specific stuff
 .idea/
 .vscode/
+.DS_Store
+yarn-error.log
 
 # Generated proto files
 web-app/src/proto/*
 !web-app/src/proto/.gitkeep
 api/proto/*
 !api/proto/.gitkeep
 
+# Un-vendored dependencies
+web-app/node_modules/
```

Technically, any `node_modules` folder anywhere in the repo should probably be ignored
```suggestion
node_modules
```
which would also remove the necessity for ignoring `web-app/src/node_modules/@sapper/`